README.md

```diff
@@ -184,7 +184,34 @@ Example:
 ```
 
 See this [example `<SelectionDot />` component](./example/src/components/CustomSelectionDot.tsx).
+### `events`
+  An array of events to be marked in the graph. The position is calculated based on the `date` property of each event relatively to `points` of the graph.
 
+### `EventComponent`
+  A component that is used to render an event.
+
+### `EventTooltipComponent`
```

Comment:
Please create some default component EventTooltip and link it here as example (same asi SelectionDot)

Reply:
Default component created in the fixup 5ff9fd7.